backend/database/repositories/filesystem.go

```diff
@@ -14,7 +14,7 @@ type filesystemRepository struct {
 // We really should use an ORM jesus this is ugly
 func (rep filesystemRepository) query(query string, input ...interface{}) (FilesystemEntry, error) {
 	entity := FilesystemEntry{}
-	children := []int{}
+	children := []string{}
```

Comment:
Following on from my comment in `repositories/main.go` if we go with an actual UUID type then this would be turned into:

```go
children := []uuid.UUID

for rows.Next() {
   var x string
   rows.Scan(&x)

   if parsedId, err := uuid.Parse(x); err != nil {
     children = append(children, parsedId)
   } else {
     return FilesystemEntry{}, err
   }
}
```

Reply:
Currently done something like 

```go
    children := []uuid.UUID{}
	// finally scan in the rows
	for rows.Next() {
		var x uuid.UUID
		err := rows.Scan(&x)
		if err != nil {
			return FilesystemEntry{}, err
		}

		children = append(children, x)
	}
```
i.e. changed `x` to be of type `uuid.UUID` and did not include `uuid.Parse`